### ISSUE TITLE
Are "other project" types tagged properly?

### ISSUE THREAD

**Issue by brucecrevensten** [ua-snap/nccwsc-projects#121]
Investigate: search by “moose” and check if the “Other Product” tag isn’t showing up somewhere (the bubble-tag that flags if something is a project, data, etc).

**Comment by brucecrevensten** [comment]
Some items in the search results from the home page don't show the "Type" badge.  This is because the item has nothing associated with it in the "Types" field coming from ScienceBase.  Here's an example:

https://www.sciencebase.gov/catalog/item/5c522b3de4b0708288fb4460

Parent project:
https://www.sciencebase.gov/catalog/item/5876316ce4b04eac8e074722

Parent project on Project Explorer:
http://dev.cascprojects.org/#/project/5050cb0ee4b0be20bb30eac0/5876316ce4b04eac8e074722

In this case it seems that it's clearly a Publication, but I don't know that that's true in general.  Possible approaches to take here,

- Find and update these in ScienceBase so they are properly tagged,
- Add an "other" badge to these even if that may not be strictly correct,
- Assume they're all Publications and add that badge (possibly wrong),
- Remove these items from search results (probably bad).

**Comment by hollychandler** [comment]
@brucecrevensten  I just did a search for "moose" and another for "bird" and the Other Product tag (as well as the project, data, and publication tags) does seem to be working as expected. I think the tags look great in the search results. The one thing I did notice though is that the "Project Type" filter doesn't give you the option to filter by "Other Product", and I think it should. Also, I'm not sure it makes sense to have this filter be called "Project Type", since not everything is a Project. Maybe we have it be called just "Type"....or "Resource Type"? 
![image](https://user-images.githubusercontent.com/85572743/180273912-6038dcf4-784e-41cc-8b98-5d5e9d5e409c.png)
